infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutKmPostController.kt

```diff
@@ -158,8 +158,8 @@ class LayoutKmPostController(
     fun getKmLength(
         @PathVariable("publishType") publishType: PublishType,
         @PathVariable("id") kmPostId: IntId<TrackLayoutKmPost>,
-    ): TrackLayoutKmPostLength {
+    ): ResponseEntity<TrackLayoutKmPostLength> {
         logger.apiCall("getKmLength", "id" to kmPostId, "publishType" to publishType)
-        return TrackLayoutKmPostLength(kmPostService.getSingleKmPostLength(publishType, kmPostId))
+        return toResponse(kmPostService.getSingleKmPostLength(publishType, kmPostId))
```

Comment:
Tuntuu sinällään vähän hassulta palauttaa yksittäinen arvo oliomuodossa. Olen melko varma että tämän voisi palauttaa ihan vaan `Double`:na sen sijasta että tämän wräppää `TrackLayoutKmPost`:iin ja frontti osaisi silti käsitellä sitä oikein ilman sen suurempia muutoksia.

Reply:
Tällaisenaan pitää paikkansa mutta ei tuo wrappaus haittaakaan. Plus tuohon on helpompi tehdä myöhemmin lisäyksiä tarpeen mukaan. Jos näitä esim haetaan toisessa API:ssa monta kerrallaan niin olisi luontevaa lisätä olioon ID.